src/components/platformSidebar.tsx

```diff
@@ -83,7 +83,7 @@ export const PlatformSidebar = ({
       <DynamicNav
         root={`/${pathRoot}/performance`}
         title="Performance Monitoring"
-        prependLinks={[[`/${pathRoot}/performance/`, "Enable Performance"]]}
+        prependLinks={[[`/${pathRoot}/performance/`, "Enabling Tracing"]]}
```

Comment:
Tracing is the package name, but we refer to Performance Monitoring everywhere else. I'd recommend, if changing the title, change it to "Enable Performance Monitoring"

Reply:
I was purposely picking the synonym, since it sits in the sidebar immediately below the "Performance Monitoring" section title. (It's also shorter and therefore fits better.) NB that this doesn't affect the page title, which remains "Performance Monitoring." Only the sidebar is changed.